repository-data/webfiles/src/main/resources/site/scss/intranet/components/_intra-home-article-grid.scss

```diff
@@ -0,0 +1,136 @@
+// Intranet `article grid`
+.intra-home-article-grid {
+    @include clearfix;
+
+    position: relative;
+    display: block;
+    margin-bottom: 16px;
+    padding: 0;
+    list-style: none;
+    
+    &-item {
+        position: relative;
+        display: inline-block;
+        float: left;
+        width: 33.333%;
+        margin-bottom: 18px;
+
+        &:nth-child(3n+1) {
+            padding-right: 10px;
+        }
+        
+        &:nth-child(3n+2) {
+            padding-left: 5px;
+            padding-right: 5px;
+        }
+
+        &:nth-child(3n+3) {
+            padding-left: 10px;
+        }
+
+        /*@include media($max-width: $min-desktop-width) {
+            max-height: 350px;
+            width: 50%;
+            padding-left: 10px;
+            padding-right: 10px;
+        }*/
```

Comment:
Try to avoid keeping commended out code.

Reply:
I agree. I left this one in there, as depending how the real document contents render in these boxes we might need to keep a 2 column version. I don't know until we plug this into Shane's channel editor version of the home page,.